openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -306,44 +353,6 @@ def verify_fields_visible_to_all_users(response):
         response = self.send_get(client, query_parameters='view=shared')
         verify_fields_visible_to_all_users(response)
 
-    def test_get_account_default(self):
```

Comment:
Good catch- no, it shouldn't. I thought this was a test added by the original author.